tests/dualtor/test_switchover_failure.py

```diff
@@ -166,9 +167,14 @@ def test_mac_move_during_switchover(
     # Pause syncd and trigger a switchover. Since syncd is paused, orchagent will hang (if the bug is present)
     # which allows us to pause orchagent mid-switchover
     rand_selected_dut.control_process("syncd", pause=True)
-    rand_selected_dut.shell(
-        "config mux mode active {}".format(common_setup_teardown["intf1"])
-    )
+    if cable_type == CableType.active_standby:
+        rand_selected_dut.shell(
+            "config mux mode active {}".format(common_setup_teardown["intf1"])
```

Comment:
are we already toggled to the random selected dut by fixture `toggle_all_simulator_ports_to_rand_unselected_tor`?

Reply:
yes, this line is to trigger a switchover on a single interface. 

the test was introduced for mac move causing switchover failure on some old images